src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -109,7 +109,7 @@ public UserTenantCollection getByUserId(UUID consortiumId, UUID userId, Integer
   @Transactional
   public UserTenant save(UUID consortiumId, UserTenant userTenantDto, boolean isSystemUserContextRequired) {
     log.debug("Going to save user with id: {} into tenant: {}", userTenantDto.getUserId(), userTenantDto.getTenantId());
-    FolioExecutionContext currentTenantContext = (FolioExecutionContext) folioExecutionContext.getInstance();
+    FolioExecutionContext currentTenantContext = folioExecutionContext;
```

Comment:
we should not need this variable, setting context can be done using folioExecutionContext field

Reply:
Fixed